getSoilClay.R

```diff
@@ -0,0 +1,46 @@
+
+rm(list=ls())
+
+library(rgdal)
+library(gdalUtils)
+library(sf)
+
+url = "https://files.isric.org/soilgrids/latest/data/"
+
+# variable of interest
+voi = "clay"
+depth = "15-30cm"
+quantile = "mean"
+
+variable = paste(url, voi, sep="")
```

Comment:
FYI there's a `paste0` function that's simpler to use.

Reply:
file no longer exsists